holoviews/operation/element.py

```diff
@@ -1218,3 +1219,71 @@ def _process(self, p, element, ranges=None):
                                   datatype=[default_datatype])
             data[(d1.name, d2.name)] = el
         return data
+
+
+class dendrogram(Operation):
+    """The dendrogram operation computes one or two adjoint dendrogram of the
+    data along the specified dimension(s). The operation uses the scipy
+    dendrogram algorithm to compute the tree structure of the data. The
+    operation is typically used to visualize hierarchical clustering of the
+    data.
+    """
+
```

Comment:
Maybe add `adjoined` as a parameter and if it's false just return the Dendrogram itself.

Reply:
Added in https://github.com/holoviz/holoviews/pull/6511/commits/58de7aa58293aa50dafdbed56be0b5d5b7540e8d